lib/nexpose/connection.rb

```diff
@@ -76,23 +78,23 @@ def self.from_uri(uri, user, pass, silo_id = nil, token = nil, trust_cert = nil)
     # @param [String] token The two-factor authentication (2FA) token for Nexpose sessions.
     # @param [String] trust_cert The PEM-formatted web certificate of the Nexpose console. Used for SSL validation.
     def initialize(ip, user, pass, port = 3780, silo_id = nil, token = nil, trust_cert = nil)
-      @host = ip
-      @port = port
-      @username = user
-      @password = pass
-      @token = token
-      @silo_id = silo_id
-      unless trust_cert.nil?
```

Comment:
Sorry to nit pick this is another case where code inspection is not friendly to debugging, by making this a single line call you can lose break point granularity.

Reply:
can you give an example of what you mean? are you saying you'd want to put a breakdown on the inside the `unless` statement? if that's the case, you'd be able to just put the breakpoint inside the `create_trust_store` method.

Reply:
Again is just a convenience thing to me being able to break when `trust_cert.nil?` without setting a conditional on the break point that is already in the code and see `@trust_store` before and after the call to `create_trust_store(trust_cert)`

In this case it may not be all that useful since `create_trust_store` is local code, I am just adding context about some things that rubocop thinks are bad that do little to impact performance but can help debugging and readability.  Especially if the method called is from some utility library that is used heavily.

Reply:
Fair enough, I see what you're saying. Personally, I am using breakpoints with `binding.pry` so for your example I would probably do `binding.pry if trust_cert.nil?` on a new line. Where I am assuming if we were to talk about "clicking an IDE" to create a breakpoint, that's what you're referring to. 

That's a fair enough concern, and I'll keep it in mind going forward. On this particular line, do you have a preference if we leave it as is, or should I change it back?

Reply:
This is fine to leave as is.